src/main/java/seedu/exercise/ui/MainWindow.java

```diff
@@ -147,6 +154,21 @@ void fillInnerParts() {
         suggestionPanelPlaceholder.getChildren().add(suggestionPanel.getRoot());
     }
 
+    /**
+     * Sets the chart type in chartPlaceholder according to user input.
+     */
+    private void setChart() {
+        Statistic statistic = logic.getStatistic();
+        String chart = statistic.getChart();
+        chartPlaceholder.getChildren().clear();
+        if (chart.equals("barchart")) {
+            chartPlaceholder.getChildren().add(new BarChartPanel(statistic).getRoot());
+        } else if (chart.equals("linechart")) {
+            chartPlaceholder.getChildren().add(new LineChartPanel(statistic).getRoot());
+        } else {
+            chartPlaceholder.getChildren().add(new PieChartPanel(statistic).getRoot());
```

Comment:
else if (chart.equals("piechart")) {

}  else {
     // Default chart / some text to show chart display error
}

Reply:
done